+++
title = "Taxonomic classification methodf or metagenomics based on core protein families with Core-Kaiju"
date = "2020-09-18"

authors = ["Anna Tovo", "Peter Menzel", "Anders Krogh", "Marco Cosentino Lagomarsino", "Samir Suweis"]

publication_types = ["2"]

publication = "Nucleic Acids Research"
publication_short = ""

abstract = " "
image_preview = ""

featured = true

projects = ["QuCoBiM"]

url_custom = [{name = "Website", url = "https://doi.org/10.1093/nar/gkaa568"}]

doi = "10.1093/nar/gkaa568"

+++
# Abstract
Characterizing species diversity and composition of bacteria hosted by biota is revolutionizing our understanding of the role of symbiotic interactions in ecosystems. Determining microbiomes diversity implies the assignment of individual reads to taxa by comparison to reference databases. Although computational methods aimed at identifying the microbe(s) taxa are available, it is well known that inferences using different methods can vary widely depending on various biases. In this study, we first apply and compare different bioinformatics methods based on 16S ribosomal RNA gene and shotgun sequencing to three mock communities of bacteria, of which the compositions are known. We show that none of these methods can infer both the true number of taxa and their abundances. We thus propose a novel approach, named Core-Kaiju, which combines the power of shotgun metagenomics data with a more focused marker gene classification method similar to 16S, but based on emergent statistics of core protein domain families. We thus test the proposed method on various mock communities and we show that Core-Kaiju reliably predicts both number of taxa and abundances. Finally, we apply our method on human gut samples, showing how Core-Kaiju may give more accurate ecological characterization and a fresh view on real microbiomes.
